backend/src/main/kotlin/com/example/backend/resolver/MutationResolver.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.backend.resolver
+
+import com.example.backend.utils.CalculadoraPrimos.Companion.contarPrimos
+import org.springframework.graphql.data.method.annotation.MutationMapping
+import org.springframework.stereotype.Controller
+
+data class Input(val number: Int)
+data class Result(val numeroPrimos: Int, val tempoDeCalculo: Int)
+
+@Controller
+class MutationResolver() {
+
+    @MutationMapping
+    fun processNumber(input: Input): Result {
```

Comment:
Como essa operação não faz nenhuma alteração de dado no servidor, faria mais sentido ser uma query 

Esse [artigo](https://www.apollographql.com/blog/mutation-vs-query-when-to-use-graphql-mutation) explica bem a diferença entre as duas, a [doc](https://graphql.org/learn/mutations/) também pode ajudar